Hitch OSP build issue

The hitch port has an issue with not including wolfSSL configuration before using wolfSSL headers:
```
  CC       libcfg_a-cfg_lex.o
In file included from /home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/wolfcrypt/types.h:34,
                 from /home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/openssl/ssl.h:32,
                 from configuration.h:12,
                 from cfg_lex.l:5:
/home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/wolfcrypt/settings.h:336:6: error: #error "No configuration for wolfSSL detected, check header order"
  336 |     #error "No configuration for wolfSSL detected, check header order"
      |      ^~~~~
In file included from /home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/wolfcrypt/types.h:34,
                 from /home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/openssl/ssl.h:32,
                 from configuration.h:12,
                 from cfg_lex.l:5:
/home/runner/work/wolfssl/wolfssl/build-dir/include/wolfssl/wolfcrypt/settings.h:3708:14: warning: #warning "For timing resistance / side-channel attack prevention consider using harden options" [-Wcpp]
 3708 |             #warning "For timing resistance / side-channel attack prevention consider using harden options"
      |              ^~~~~~~
make[3]: *** [Makefile:784: libcfg_a-cfg_lex.o] Error 1
```
original failure report (may be expired in the future)
https://github.com/wolfSSL/wolfssl/actions/runs/12241546663/job/34147016779?pr=8262

This issue was exposed after wolfSSL PR https://github.com/wolfSSL/wolfssl/pull/8262

--- a/hitch/hitch_1.8.0.patch
+++ b/hitch/hitch_1.8.0.patch
@@ -3,7 +3,7 @@ Author: Kareem <kareem@wolfssl.com>
 Date:   Thu Mar 5 16:34:26 2026 -0700
 
     Add wolfSSL support to hitch.
-    
+
     To use it, build wolfSSL with:
     ./autogen.sh
     ./configure --enable-hitch
@@ -162,6 +162,18 @@ index e95e213..fc88cbe 100644
  
  SH_TESTS="$(cd $srcdir/src && echo tests/test*.sh)"
  AC_SUBST(SH_TESTS)
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 9051f4c..e1bc366 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -50,6 +50,7 @@ libcfg_a_SOURCES = \
+ 	cfg_parser.y
+ 
+ libcfg_a_CFLAGS = \
++	$(HITCH_CFLAGS) \
+ 	$(SSL_CFLAGS)
+ 
+ libforeign_a_SOURCES = \
 diff --git a/src/hitch.c b/src/hitch.c
 index a499c98..8f4da3a 100644
 --- a/src/hitch.c

